csm/cli/schema/alerts.json

```diff
@@ -13,7 +13,7 @@
           "dest": "duration",
           "nargs": "+",
           "type": "str",
-          "help": "Time period, for which we request alerts. Format: <x>s <y>m <z>h <q>d. Where x, y, z, q is amounts of seconds, minutes, hours, days respectively",
+          "help": "Time period, for which we request alerts. Format: <x>s <y>m <z>h <q>d. Where x, y, z, q is amounts of seconds, minutes, hours, days respectively. Past 60 Seconds Alerts will be Displayed if not set explicitly.",
```

Comment:
In every help -
Displayed -> displayed.

Reply:
Done